src/NuGet.Core/NuGet.Protocol/Resources/PackageSearchResourceV3.cs

```diff
@@ -32,14 +62,27 @@ public PackageSearchResourceV3(RawSearchResourceV3 searchResource)
         /// <returns>List of package meta data.</returns>
         public override async Task<IEnumerable<IPackageSearchMetadata>> SearchAsync(string searchTerm, SearchFilter filter, int skip, int take, Common.ILogger log, CancellationToken cancellationToken)
         {
+            IEnumerable<PackageSearchMetadata> searchResultMetadata;
             var metadataCache = new MetadataReferenceCache();
-            var searchResultMetadata = await _rawSearchResource.SearchNew(
+
+            if (_client != null && _searchEndpoints != null)
+            {
+                searchResultMetadata = await Search(
                     searchTerm,
                     filter,
                     skip,
                     take,
                     Common.NullLogger.Instance,
                     cancellationToken);
+            }
+            else
+            {
+#pragma warning disable 618
```

Comment:
Same here, use GlobalSuppressions.

Reply:
Same here VS was not offering any option to suppress this into GlobalSuppressions file. It seems other soIution is have to suppress for whole this file.

Reply:
This is CS0618, compiler warning, and cannot be suppressed using attributes. They have to be suppressed using `#pragma`s